challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
+        b: |
+          Because Vue's `v-show` directive cancels any transition rules
+          that we apply to the element.
+        c: |
+          Because Vue's `v-show` directive makes the `opacity` property
+          jump from `1` to `0` immediately.
+        d: |
+          Because Vue's `v-show` directive works on the `display` property
+          of an element, which is not the property we're transitioning.
+      correct: d
+
+extending_transitions:
+  question: |
+    Our component that shows or hides a disclaimer message upon buying
+    pizza slices has made a big impact in our customer's health! Yay!
+
+    But our CEO is not very happy with the visual results... Nay!
+
+    He now wants us to just simply fade in or out such message when
+    a user wants to buy too much pizza. We can use Vue transitions for
+    that!
+
+    Here's the initial component:
+
+    ```vue
+    <template>
+        <div>
+            <button @click="addPizzaToCart" />
+
+            <p v-show="showDisclaimer">
+                <strong>Disclaimer</strong>: Eating too much pizza can be
+                bad for your health!
+            </p>
+        </div>
+    </template>
+
+    <script>
+    import { buyPizza } from '@/services/buy-pizza';
+
+    export default {
+        name: 'PizzaCheckout',
+        data() {
+            return {
+                showDisclaimer: false,
+            };
+        },
+        methods: {
+            async addPizzaToCart() {
+                await buyPizza();
+            }
+        },
+    };
+    </script>
+    ```
+
+    How can we make it work using Vue transitions?
+  explanation: |
+    Using Vue transitions can seem a bit complicated at first, but it's
+    really a very usefull feature once you get to know it!
+
+    Remember the classes it relies on have to be under the `:global` keyword,
+    otrherwise your transition won't work where the class names have been
+    altered!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```vue
+          <template>
+              <div :class="$style.component">
+                  <button @click="addPizzaToCart" />
+
+                  <transition>
+                      <p v-show="showDisclaimer">
+                          <strong>Disclaimer</strong>: Eating too much pizza can be
+                          bad for your health!
+                      </p>
+                  </transition>
+              </div>
+          </template>
+
+          <script>
+          import { buyPizza } from '@/services/buy-pizza';
+
+          export default {
+              name: 'PizzaCheckout',
+              data() {
+                  return {
+                      showDisclaimer: false,
+                  };
+              },
+              methods: {
+                  async addPizzaToCart() {
+                      await buyPizza();
+                  }
+              },
+          };
+          </script>
```

Comment:
Let's not include the `script` part on any of these. We need to help the user focus in on the exact changes that we're asking them about. Then, to make it more clear, I'd split into 2 different \`\`\` code blocks - one for the template and the other for styles.